packages/core/apps/core-apps-browser-mocks/kibana.jsonc

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I vaguely remember dev only packages here had issues, is that still the case?

Reply:
This package doesn't have `devOnly` defined, though I think it would make sense to do so.

Reply:
Most (all?) of our mock packages are currently missing the `devOnly` flag. We were planning on checking everything during the cleanup phase of Core's package migration